developer/src/common/web/utils/src/developer-utils-messages.ts

```diff
@@ -58,4 +58,22 @@ export class DeveloperUtilsMessages {
     this.ERROR_InvalidPackageFile,
     `Package source file is invalid: ${(o.e ?? 'unknown error').toString()}`
   );
+
+  static ERROR_InvalidProjectFile = SevError | 0x000A;
+  static Error_InvalidProjectFile = (o:{message:string}) => m(
+    this.ERROR_InvalidProjectFile,
+    `Project file is not valid: ${def(o.message)}`,
+  );
+
+  static ERROR_UnsupportedProjectVersion = SevError | 0x0013;
```

Comment:
Any reason to jump from 0x000A to 0x0013?

Reply:
Good catch, forgot to adjust, will go to B,C 

```suggestion
  static ERROR_UnsupportedProjectVersion = SevError | 0x000A;
```